src/features/device-state/routes/state-v2.ts

```diff
@@ -197,13 +186,6 @@ function buildAppFromRelease(
 	};
 }
 
-// These 2 config vars below are mapped to labels if missing for backwards-compatibility
-// See: https://github.com/resin-io/hq/issues/1340
-const ConfigurationVarsToLabels = {
-	RESIN_SUPERVISOR_UPDATE_STRATEGY: 'io.resin.update.strategy',
```

Comment:
Is this being moved somewhere else, or are you removing it entirely? This is still advertised as the main way to controlled the update strategy in our docs https://www.balena.io/docs/learn/deploy/release-strategy/update-strategies/#controlling-the-update-strategy

Reply:
Ignore me, I saw the code was moved to utils